dev_scripts_helpers/github/dockerized_invite_gh_contributors.py

```diff
@@ -63,6 +59,25 @@ def extract_usernames_from_gsheet(gsheet_url: str) -> List[str]:
     return usernames
 
 
+def extract_usernames_from_csv(csv_path: str) -> List[str]:
+    """
+    Extract GitHub usernames from a CSV file containing a *GitHub user* column.
+
+    :param csv_path: path to csv
+    :return: github usernames
+    """
+    usernames: List[str] = []
+    with open(csv_path, newline="", encoding="utf-8") as csv_file:
+        reader = csv.DictReader(csv_file)
+        if not reader.fieldnames or "GitHub user" not in reader.fieldnames:
```

Comment:
You can use a hdbg.dassert_in since it has a nice message and it's more readable